src/pages/programDisaggregations/form/ProgramIndicatorMappingSection.tsx

```diff
@@ -324,11 +338,15 @@ export const CategoryMappingSelect = ({
         })
     }, [availableMappings])
 
+    const categoryMappingLabelPrefix = i18n.t(`Category mapping: `, {
```

Comment:
adding missing translation.
we need to pass a string to label prop, we could interpolate, i18n.t(`Category mapping: {{mappingName}}`, but then we need to [unescape](https://www.i18next.com/translation-function/interpolation#unescape) in case in case there is a special character (e.g. `å`, `é`, `&`). Technically, at this point the category.displayName should have been verified by the backend, so this is reasonably safe, but I don't want to do this here as the other part where we have the `Category mapping: `, we are using the mapping name, which is a user input within the app (hence we cannot trust it to be safe)